import/importer.mjs

```diff
@@ -0,0 +1,66 @@
+import { readFileSync } from 'fs';
+import { spawn } from 'child_process';
+import markdownConverter from "./markdown-converter.mjs";
+import { exit } from 'process';
+import { create } from 'domain';
+
+export default class Importer {
+    static processNotifyTemplate = (filename) => {
+        let rawdata = readFileSync(filename);
+        let contents = JSON.parse(rawdata);
+    
+        let i = 0
+        contents.forEach(item => {
+            
+            const convertedBody = markdownConverter(item.content)
```

Comment:
Looks like `const convertedBody = markdownConverter(item.content)` is not in use